[2024-01-05T01:17:25.821+0000] {processor.py:161} INFO - Started process (PID=23724) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:17:25.822+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T01:17:25.825+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:17:25.824+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:17:25.833+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:17:25.833+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T01:17:25.833+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:17:25.833+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T01:17:25.834+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:17:25.850+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2024-01-05T01:18:46.031+0000] {processor.py:161} INFO - Started process (PID=24450) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:18:46.033+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T01:18:46.035+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:18:46.035+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:18:46.038+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:18:46.038+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T01:18:46.038+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:18:46.038+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T01:18:46.039+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:18:46.052+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2024-01-05T01:23:00.838+0000] {processor.py:161} INFO - Started process (PID=27912) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:23:00.866+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T01:23:00.867+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:23:00.866+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:23:00.871+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:23:00.870+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T01:23:00.877+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:23:00.874+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T01:23:00.877+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:23:00.888+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.054 seconds
[2024-01-05T01:28:01.021+0000] {processor.py:161} INFO - Started process (PID=32827) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:28:01.037+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T01:28:01.038+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:28:01.037+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:28:01.064+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:28:01.062+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T01:28:01.065+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:28:01.065+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T01:28:01.065+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:28:01.079+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.062 seconds
[2024-01-05T01:28:56.308+0000] {processor.py:161} INFO - Started process (PID=32990) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:28:56.317+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T01:28:56.320+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:28:56.320+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:28:56.329+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:28:56.326+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T01:28:56.329+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:28:56.329+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T01:28:56.329+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:28:56.457+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.155 seconds
[2024-01-05T01:29:52.966+0000] {processor.py:161} INFO - Started process (PID=33154) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:29:52.973+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T01:29:52.975+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:29:52.974+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:29:52.978+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:29:52.978+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T01:29:52.979+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:29:52.979+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T01:29:52.979+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:29:52.989+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2024-01-05T01:30:43.708+0000] {processor.py:161} INFO - Started process (PID=33304) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:30:43.711+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T01:30:43.712+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:30:43.712+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:30:43.715+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:30:43.714+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T01:30:43.715+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:30:43.715+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T01:30:43.716+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:30:43.726+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.020 seconds
[2024-01-05T01:31:39.564+0000] {processor.py:161} INFO - Started process (PID=33462) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:31:39.596+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T01:31:39.597+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:31:39.597+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:31:39.600+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:31:39.600+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T01:31:39.605+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:31:39.604+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T01:31:39.605+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:31:39.639+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.082 seconds
[2024-01-05T01:32:33.915+0000] {processor.py:161} INFO - Started process (PID=33744) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:32:33.925+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T01:32:33.925+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:32:33.925+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:32:33.936+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:32:33.934+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T01:32:33.936+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:32:33.936+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T01:32:33.937+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:32:33.965+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.052 seconds
[2024-01-05T01:33:46.783+0000] {processor.py:161} INFO - Started process (PID=33936) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:33:46.789+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T01:33:46.790+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:33:46.789+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:33:46.796+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:33:46.794+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T01:33:46.797+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:33:46.796+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T01:33:46.797+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:33:46.816+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2024-01-05T01:34:41.168+0000] {processor.py:161} INFO - Started process (PID=34108) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:34:41.202+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T01:34:41.204+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:34:41.204+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:34:41.211+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:34:41.210+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T01:34:41.211+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:34:41.211+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T01:34:41.211+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:34:41.221+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.057 seconds
[2024-01-05T01:35:37.569+0000] {processor.py:161} INFO - Started process (PID=34263) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:35:37.579+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T01:35:37.580+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:35:37.579+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:35:37.583+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:35:37.582+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T01:35:37.583+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:35:37.583+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T01:35:37.583+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:35:37.593+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2024-01-05T01:36:30.886+0000] {processor.py:161} INFO - Started process (PID=34481) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:36:30.898+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T01:36:30.901+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:36:30.898+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:36:30.909+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:36:30.909+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T01:36:30.909+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:36:30.909+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T01:36:30.910+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:36:30.964+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.081 seconds
[2024-01-05T01:37:23.247+0000] {processor.py:161} INFO - Started process (PID=34657) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:37:23.256+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T01:37:23.257+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:37:23.257+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:37:23.260+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:37:23.260+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T01:37:23.260+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:37:23.260+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T01:37:23.261+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:37:23.269+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2024-01-05T01:38:14.764+0000] {processor.py:161} INFO - Started process (PID=34867) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:38:14.775+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T01:38:14.775+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:38:14.775+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:38:14.778+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:38:14.778+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T01:38:14.779+0000] {logging_mixin.py:188} INFO - [2024-01-05T01:38:14.779+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T01:38:14.779+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T01:38:14.788+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-01-05T15:16:19.101+0000] {processor.py:161} INFO - Started process (PID=4580) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:16:19.113+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T15:16:19.115+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:16:19.114+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:16:19.121+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:16:19.120+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T15:16:19.123+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:16:19.123+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T15:16:19.123+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:16:19.139+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2024-01-05T15:17:28.708+0000] {processor.py:161} INFO - Started process (PID=5249) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:17:28.717+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T15:17:28.739+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:17:28.721+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:17:28.812+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:17:28.789+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T15:17:28.813+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:17:28.812+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T15:17:28.817+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:17:28.912+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.242 seconds
[2024-01-05T15:20:43.248+0000] {processor.py:161} INFO - Started process (PID=7715) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:20:43.260+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T15:20:43.263+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:20:43.261+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:20:43.271+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:20:43.270+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T15:20:43.272+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:20:43.272+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T15:20:43.273+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:20:43.290+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.046 seconds
[2024-01-05T15:21:38.479+0000] {processor.py:161} INFO - Started process (PID=7821) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:21:38.489+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T15:21:38.490+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:21:38.490+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:21:38.496+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:21:38.495+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T15:21:38.497+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:21:38.496+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T15:21:38.498+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:21:38.514+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.040 seconds
[2024-01-05T15:22:33.396+0000] {processor.py:161} INFO - Started process (PID=7941) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:22:33.406+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T15:22:33.408+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:22:33.407+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:22:33.413+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:22:33.412+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T15:22:33.413+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:22:33.413+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T15:22:33.414+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:22:33.428+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.037 seconds
[2024-01-05T15:23:24.186+0000] {processor.py:161} INFO - Started process (PID=8080) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:23:24.194+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T15:23:24.196+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:23:24.195+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:23:24.201+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:23:24.201+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T15:23:24.202+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:23:24.202+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T15:23:24.202+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:23:24.220+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.040 seconds
[2024-01-05T15:24:19.591+0000] {processor.py:161} INFO - Started process (PID=8219) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:24:19.601+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T15:24:19.603+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:24:19.602+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:24:19.609+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:24:19.608+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T15:24:19.610+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:24:19.610+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T15:24:19.610+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:24:19.625+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.038 seconds
[2024-01-05T15:25:11.351+0000] {processor.py:161} INFO - Started process (PID=8353) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:25:11.362+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T15:25:11.364+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:25:11.363+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:25:11.370+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:25:11.370+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T15:25:11.371+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:25:11.371+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T15:25:11.371+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:25:11.388+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.041 seconds
[2024-01-05T15:26:03.483+0000] {processor.py:161} INFO - Started process (PID=8553) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:26:03.492+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T15:26:03.494+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:26:03.493+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:26:03.501+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:26:03.500+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T15:26:03.503+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:26:03.502+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T15:26:03.503+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:26:03.519+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.043 seconds
[2024-01-05T15:27:18.264+0000] {processor.py:161} INFO - Started process (PID=8788) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:27:18.273+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T15:27:18.275+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:27:18.274+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:27:18.282+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:27:18.281+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T15:27:18.283+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:27:18.283+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T15:27:18.283+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:27:18.300+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.040 seconds
[2024-01-05T15:28:08.721+0000] {processor.py:161} INFO - Started process (PID=8927) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:28:08.735+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T15:28:08.737+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:28:08.736+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:28:08.744+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:28:08.743+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T15:28:08.745+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:28:08.744+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T15:28:08.746+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:28:08.766+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.046 seconds
[2024-01-05T15:28:59.547+0000] {processor.py:161} INFO - Started process (PID=9060) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:28:59.557+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T15:28:59.558+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:28:59.557+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:28:59.571+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:28:59.569+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T15:28:59.571+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:28:59.571+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T15:28:59.572+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:28:59.588+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2024-01-05T15:29:52.733+0000] {processor.py:161} INFO - Started process (PID=9336) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:29:52.736+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T15:29:52.737+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:29:52.737+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:29:52.742+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:29:52.741+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T15:29:52.743+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:29:52.743+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T15:29:52.743+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:29:52.759+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.031 seconds
[2024-01-05T15:30:45.099+0000] {processor.py:161} INFO - Started process (PID=9530) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:30:45.108+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T15:30:45.111+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:30:45.110+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:30:45.125+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:30:45.125+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T15:30:45.127+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:30:45.127+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T15:30:45.128+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:30:45.143+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.050 seconds
[2024-01-05T15:31:37.427+0000] {processor.py:161} INFO - Started process (PID=9606) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:31:37.437+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T15:31:37.439+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:31:37.438+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:31:37.465+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:31:37.461+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T15:31:37.469+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:31:37.466+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T15:31:37.478+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:31:37.500+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.080 seconds
[2024-01-05T15:32:26.872+0000] {processor.py:161} INFO - Started process (PID=9800) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:32:26.875+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T15:32:26.877+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:32:26.876+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:32:26.885+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:32:26.884+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T15:32:26.885+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:32:26.885+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T15:32:26.887+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:32:26.936+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.070 seconds
[2024-01-05T15:33:41.305+0000] {processor.py:161} INFO - Started process (PID=10004) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:33:41.314+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T15:33:41.316+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:33:41.315+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:33:41.323+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:33:41.322+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T15:33:41.323+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:33:41.323+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T15:33:41.324+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:33:41.342+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.042 seconds
[2024-01-05T15:34:55.657+0000] {processor.py:161} INFO - Started process (PID=10211) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:34:55.667+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T15:34:55.669+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:34:55.668+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:34:55.678+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:34:55.676+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T15:34:55.679+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:34:55.679+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T15:34:55.679+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:34:55.694+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.042 seconds
[2024-01-05T15:35:45.616+0000] {processor.py:161} INFO - Started process (PID=10468) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:35:45.626+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T15:35:45.628+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:35:45.628+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:35:45.637+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:35:45.635+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T15:35:45.637+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:35:45.637+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T15:35:45.638+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:35:45.654+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
[2024-01-05T15:36:39.557+0000] {processor.py:161} INFO - Started process (PID=10606) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:36:39.567+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T15:36:39.568+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:36:39.568+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:36:39.574+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:36:39.573+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T15:36:39.575+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:36:39.574+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T15:36:39.575+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:36:39.589+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2024-01-05T15:37:31.543+0000] {processor.py:161} INFO - Started process (PID=10744) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:37:31.553+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T15:37:31.555+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:37:31.554+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:37:31.560+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:37:31.559+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T15:37:31.560+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:37:31.560+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T15:37:31.561+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:37:31.576+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.036 seconds
[2024-01-05T15:38:20.273+0000] {processor.py:161} INFO - Started process (PID=10873) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:38:20.283+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T15:38:20.285+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:38:20.284+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:38:20.293+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:38:20.292+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T15:38:20.293+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:38:20.293+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T15:38:20.294+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:38:20.308+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.040 seconds
[2024-01-05T15:39:33.311+0000] {processor.py:161} INFO - Started process (PID=11039) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:39:33.329+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T15:39:33.332+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:39:33.331+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:39:33.340+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:39:33.339+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T15:39:33.341+0000] {logging_mixin.py:188} INFO - [2024-01-05T15:39:33.340+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T15:39:33.341+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T15:39:33.360+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.053 seconds
[2024-01-05T16:31:43.796+0000] {processor.py:161} INFO - Started process (PID=4977) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:31:43.805+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T16:31:43.806+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:31:43.806+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:31:43.811+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:31:43.811+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T16:31:43.812+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:31:43.812+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T16:31:43.812+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:31:43.821+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-01-05T16:33:10.491+0000] {processor.py:161} INFO - Started process (PID=5953) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:33:10.544+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T16:33:10.545+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:33:10.545+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:33:10.549+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:33:10.548+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T16:33:10.549+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:33:10.549+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T16:33:10.549+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:33:10.584+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.101 seconds
[2024-01-05T16:38:43.124+0000] {processor.py:161} INFO - Started process (PID=7108) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:38:43.148+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T16:38:43.149+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:38:43.149+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:38:43.152+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:38:43.151+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T16:38:43.152+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:38:43.152+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T16:38:43.153+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:38:43.162+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.044 seconds
[2024-01-05T16:39:37.692+0000] {processor.py:161} INFO - Started process (PID=7249) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:39:37.715+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T16:39:37.718+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:39:37.716+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:39:37.722+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:39:37.721+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T16:39:37.722+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:39:37.722+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T16:39:37.722+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:39:37.743+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.053 seconds
[2024-01-05T16:40:44.878+0000] {processor.py:161} INFO - Started process (PID=7741) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:40:44.885+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T16:40:44.886+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:40:44.886+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:40:44.890+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:40:44.889+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T16:40:44.890+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:40:44.890+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T16:40:44.890+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:40:44.900+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2024-01-05T16:41:36.975+0000] {processor.py:161} INFO - Started process (PID=7894) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:41:36.984+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T16:41:36.985+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:41:36.985+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:41:36.995+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:41:36.992+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T16:41:36.998+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:41:36.998+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T16:41:36.999+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:41:37.023+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.063 seconds
[2024-01-05T16:42:36.171+0000] {processor.py:161} INFO - Started process (PID=8075) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:42:36.175+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T16:42:36.176+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:42:36.176+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:42:36.182+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:42:36.181+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T16:42:36.182+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:42:36.182+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T16:42:36.182+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:42:36.191+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.024 seconds
[2024-01-05T16:43:29.823+0000] {processor.py:161} INFO - Started process (PID=8356) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:43:29.830+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T16:43:29.831+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:43:29.831+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:43:29.848+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:43:29.847+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T16:43:29.851+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:43:29.851+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T16:43:29.859+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:43:29.870+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.049 seconds
[2024-01-05T16:44:28.114+0000] {processor.py:161} INFO - Started process (PID=8514) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:44:28.129+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T16:44:28.130+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:44:28.130+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:44:28.135+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:44:28.135+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T16:44:28.142+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:44:28.140+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T16:44:28.147+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:44:28.200+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.089 seconds
[2024-01-05T16:52:57.310+0000] {processor.py:161} INFO - Started process (PID=16552) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:52:57.313+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T16:52:57.314+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:52:57.313+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:52:57.319+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:52:57.318+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T16:52:57.319+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:52:57.319+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T16:52:57.319+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:52:57.334+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-01-05T16:53:57.039+0000] {processor.py:161} INFO - Started process (PID=16776) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:53:57.047+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T16:53:57.048+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:53:57.048+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:53:57.051+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:53:57.051+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T16:53:57.052+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:53:57.052+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T16:53:57.052+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:53:57.062+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-01-05T16:54:53.994+0000] {processor.py:161} INFO - Started process (PID=16945) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:54:54.008+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T16:54:54.011+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:54:54.010+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:54:54.016+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:54:54.016+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T16:54:54.020+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:54:54.020+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T16:54:54.026+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:54:54.055+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.065 seconds
[2024-01-05T16:55:54.659+0000] {processor.py:161} INFO - Started process (PID=17118) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:55:54.668+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T16:55:54.675+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:55:54.674+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:55:54.684+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:55:54.684+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T16:55:54.687+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:55:54.687+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T16:55:54.692+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:55:54.834+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.178 seconds
[2024-01-05T16:56:54.808+0000] {processor.py:161} INFO - Started process (PID=17290) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:56:54.824+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T16:56:54.832+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:56:54.831+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:56:54.836+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:56:54.836+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T16:56:54.840+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:56:54.840+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T16:56:54.840+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:56:54.859+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.064 seconds
[2024-01-05T16:58:02.179+0000] {processor.py:161} INFO - Started process (PID=17470) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:58:02.187+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T16:58:02.189+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:58:02.188+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:58:02.194+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:58:02.192+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T16:58:02.194+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:58:02.194+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T16:58:02.194+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:58:02.210+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2024-01-05T16:59:01.075+0000] {processor.py:161} INFO - Started process (PID=17697) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:59:01.095+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T16:59:01.096+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:59:01.095+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:59:01.108+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:59:01.108+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T16:59:01.109+0000] {logging_mixin.py:188} INFO - [2024-01-05T16:59:01.109+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T16:59:01.109+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T16:59:01.128+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.059 seconds
[2024-01-05T17:00:02.212+0000] {processor.py:161} INFO - Started process (PID=17863) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:00:02.221+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T17:00:02.223+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:00:02.223+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:00:02.226+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:00:02.226+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T17:00:02.226+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:00:02.226+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T17:00:02.227+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:00:02.235+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.026 seconds
[2024-01-05T17:01:11.098+0000] {processor.py:161} INFO - Started process (PID=18047) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:01:11.124+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T17:01:11.127+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:01:11.127+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:01:11.131+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:01:11.131+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T17:01:11.131+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:01:11.131+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T17:01:11.132+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:01:11.154+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.060 seconds
[2024-01-05T17:02:11.966+0000] {processor.py:161} INFO - Started process (PID=18224) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:02:11.975+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T17:02:11.982+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:02:11.976+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:02:12.023+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:02:12.020+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T17:02:12.028+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:02:12.028+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T17:02:12.028+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:02:12.050+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.089 seconds
[2024-01-05T17:03:12.899+0000] {processor.py:161} INFO - Started process (PID=18381) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:03:12.934+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T17:03:12.935+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:03:12.935+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:03:12.943+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:03:12.943+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T17:03:12.952+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:03:12.952+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T17:03:12.957+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:03:12.984+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.093 seconds
[2024-01-05T17:04:12.530+0000] {processor.py:161} INFO - Started process (PID=18623) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:04:12.533+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T17:04:12.534+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:04:12.534+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:04:12.538+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:04:12.538+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T17:04:12.538+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:04:12.538+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T17:04:12.539+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:04:12.558+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2024-01-05T17:05:05.386+0000] {processor.py:161} INFO - Started process (PID=18780) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:05:05.387+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T17:05:05.390+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:05:05.390+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:05:05.396+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:05:05.395+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T17:05:05.396+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:05:05.396+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T17:05:05.396+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:05:05.412+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.029 seconds
[2024-01-05T17:06:10.457+0000] {processor.py:161} INFO - Started process (PID=18956) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:06:10.466+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T17:06:10.467+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:06:10.467+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:06:10.471+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:06:10.471+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T17:06:10.471+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:06:10.471+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T17:06:10.471+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:06:10.481+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.027 seconds
[2024-01-05T17:07:03.795+0000] {processor.py:161} INFO - Started process (PID=19112) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:07:03.808+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T17:07:03.811+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:07:03.811+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:07:03.817+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:07:03.816+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T17:07:03.817+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:07:03.817+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T17:07:03.817+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:07:03.831+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.039 seconds
[2024-01-05T17:08:08.994+0000] {processor.py:161} INFO - Started process (PID=19294) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:08:09.002+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T17:08:09.003+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:08:09.002+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:08:09.006+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:08:09.005+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T17:08:09.006+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:08:09.006+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T17:08:09.006+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:08:09.023+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.033 seconds
[2024-01-05T17:37:02.473+0000] {processor.py:161} INFO - Started process (PID=20501) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:37:02.475+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T17:37:02.476+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:37:02.476+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:37:02.481+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:37:02.480+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T17:37:02.490+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:37:02.490+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T17:37:02.491+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:37:03.499+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 1.031 seconds
[2024-01-05T17:38:00.440+0000] {processor.py:161} INFO - Started process (PID=20905) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:38:00.454+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T17:38:00.456+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:38:00.456+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:38:00.465+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:38:00.465+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T17:38:00.466+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:38:00.466+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T17:38:00.466+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:38:00.475+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.087 seconds
[2024-01-05T17:38:48.052+0000] {processor.py:161} INFO - Started process (PID=21104) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:38:48.055+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T17:38:48.056+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:38:48.055+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:38:48.059+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:38:48.059+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T17:38:48.059+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:38:48.059+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T17:38:48.060+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:38:48.068+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.018 seconds
[2024-01-05T17:39:42.141+0000] {processor.py:161} INFO - Started process (PID=21255) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:39:42.150+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T17:39:42.150+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:39:42.150+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:39:42.154+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:39:42.153+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T17:39:42.154+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:39:42.154+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T17:39:42.154+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:39:42.172+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.035 seconds
[2024-01-05T17:50:57.046+0000] {processor.py:161} INFO - Started process (PID=32459) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:50:57.054+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T17:50:57.055+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:50:57.054+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:50:57.066+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:50:57.065+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T17:50:57.075+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:50:57.075+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T17:50:57.075+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:50:57.088+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.045 seconds
[2024-01-05T17:51:47.669+0000] {processor.py:161} INFO - Started process (PID=32614) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:51:47.678+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T17:51:47.679+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:51:47.679+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:51:47.682+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:51:47.682+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T17:51:47.683+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:51:47.682+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T17:51:47.683+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:51:47.692+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.025 seconds
[2024-01-05T17:52:55.599+0000] {processor.py:161} INFO - Started process (PID=32860) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:52:55.603+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T17:52:55.604+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:52:55.604+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:52:55.607+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:52:55.607+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T17:52:55.607+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:52:55.607+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T17:52:55.608+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:52:55.619+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.022 seconds
[2024-01-05T17:53:54.788+0000] {processor.py:161} INFO - Started process (PID=33046) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:53:54.817+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T17:53:54.818+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:53:54.818+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:53:54.822+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:53:54.822+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T17:53:54.823+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:53:54.823+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T17:53:54.823+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:53:54.847+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.065 seconds
[2024-01-05T17:54:54.454+0000] {processor.py:161} INFO - Started process (PID=33199) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:54:54.462+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T17:54:54.463+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:54:54.463+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:54:54.467+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:54:54.466+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T17:54:54.467+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:54:54.467+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T17:54:54.467+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:54:54.477+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.028 seconds
[2024-01-05T17:55:58.644+0000] {processor.py:161} INFO - Started process (PID=33401) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:55:58.666+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T17:55:58.667+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:55:58.666+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:55:58.672+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:55:58.672+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T17:55:58.684+0000] {logging_mixin.py:188} INFO - [2024-01-05T17:55:58.684+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T17:55:58.684+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T17:55:58.715+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.076 seconds
[2024-01-05T18:00:14.678+0000] {processor.py:161} INFO - Started process (PID=33927) to work on /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T18:00:14.688+0000] {processor.py:830} INFO - Processing file /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py for tasks to queue
[2024-01-05T18:00:14.691+0000] {logging_mixin.py:188} INFO - [2024-01-05T18:00:14.691+0000] {dagbag.py:538} INFO - Filling up the DagBag from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T18:00:14.699+0000] {logging_mixin.py:188} INFO - [2024-01-05T18:00:14.698+0000] {example_local_kubernetes_executor.py:39} WARNING - Could not import DAGs in example_local_kubernetes_executor.py
Traceback (most recent call last):
  File "/home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py", line 37, in <module>
    from kubernetes.client import models as k8s
ModuleNotFoundError: No module named 'kubernetes'
[2024-01-05T18:00:14.703+0000] {logging_mixin.py:188} INFO - [2024-01-05T18:00:14.703+0000] {example_local_kubernetes_executor.py:40} WARNING - Install Kubernetes dependencies with: pip install apache-airflow[cncf.kubernetes]
[2024-01-05T18:00:14.703+0000] {processor.py:842} WARNING - No viable dags retrieved from /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py
[2024-01-05T18:00:14.729+0000] {processor.py:183} INFO - Processing /home/eder/Desktop/Airflow/Twitter/venv/lib/python3.9/site-packages/airflow/example_dags/example_local_kubernetes_executor.py took 0.062 seconds
